.github/workflows/typos.toml

```diff
@@ -23,6 +23,7 @@ EDE = "EDE"
 # HELO is an SMTP command
 HELO = "HELO"
 LKE = "LKE"
+byt = "byt"
```

Comment:
We call some variables `byt` beacuse we can't use `byte` so this is fine.

Reply:
Thank you, I should have left this as a comment.